### ISSUE TITLE
Headers remain visible as users scroll through tables/data

### ISSUE THREAD

**Issue by mhuso** [ddalthorp/GenEst#20]
Headers move out of sight as a user scrolls down a table. Can we keep them visible?
Also... bottom rectangles are apparently filters for the columns. Need something to indicate this to the user. They don't quite work as I expected them to. If I filter on something, then x out the filter, the filter isn't removed. It's only removed if I use my delete button.
The user has to know what possible values there are in order to filter them. Is there a way of having the bottom rectangles be a menu of possible values in that column (similar to excel)?

**Comment by DAPPERstats** [comment]
I've updated the data tables a bit with the DT package. They now have a search set up at the top, not filters at the bottom.
As for headers moving out of sight, that's a pretty standard situation for embedded tables like this, I don't think we can get around it aside from having fewer entries per page, so there's no scrolling down. I've set the entries at 10 per page so it's a decent stopgap for now.
This is updated in v 0.0.0.5 and will be pushed shortly.
